frog/imports/activityTypes.js

```diff
@@ -36,3 +44,19 @@ export const activityRunners = entries(activityRunnersRawInternal).reduce(
   (acc, [k, v]) => ({ ...acc, [k.split('/')[2]]: v.default }),
   activityRunnersExt
 );
+
+const internalLIs = importAll.sync('./internalLearningItems/*/index.js');
+
+const packageLIs = [...activityTypes, ...operatorTypes].reduce(
+  (acc, x) => acc.concat(x.LearningItems || []),
+  []
+);
+
+export const learningItemTypesObj: {
```

Comment:
We have the two files `opetratorTypes.js` and `activityTypes.js`. Don't you want to export this object from a new file `learningItemTypes.js`. Just to help with the logic of the code.

Reply:
Somehow I can't because it is depending on both ActivityTypes and OperatorTypes - something like a circular dependency? I tried before... Maybe it would work now, I could try again.

Reply:
I just tried again, and verified that it does not work - activityTypes is imported into learningItemTypes before it is defined, and crashes. I'm leaving this alone for now.